### PR TITLE
Implemented genre redirection 

### PR DESCRIPTION
**Pull request by solomiia-sahala**
Implemented genre redirection from details page.

### CODE REVIEW

File: src/components/MovieDetails.tsx [hunk 1]
```diff
@@ -93,10 +94,10 @@ const MovieDetails = ({ match }: MovieDetailsProps) => {
           </div>
         </div>
       </section>
-      <div className="wrapper-space">
-        <div className="genres">
-          {genres?.map(genre => <div key={genre.id}>{genre.name}</div>)}
-        </div>
+      <div className="genres">
+        {!!genres?.length && genres.map(genre => <div key={genre.id}>
```
Comment (oleksandrstarov) [inline src/components/MovieDetails.tsx:98]: !!genres?.length && genres.map( >> genres?.map(

File: src/components/RenderResults.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const RenderResults = ({ list }: RenderResultsArrList) => {
         )}
       </div>
       <div className="delivery__container">
-        {list.map((item: MovieCard) => {
+        {!!list.length ? list.map((item: MovieCard) => {
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:41]: Same as above

**Review by oleksandrstarov — APPROVED** [review]
Looks good, check comments please

**Review by di-shu — APPROVED** [review]

**Comment by oleksandrstarov** [comment]
Looks like this commit broke build on heroku https://chillzone-sigma.herokuapp.com/ 
As a rule always check that checks are passed before merging - this case commit quality failed

File: package.json [hunk 3]
```diff
@@ -60,6 +59,7 @@
   },
   "devDependencies": {
     "@types/enzyme": "^3.10.8",
-    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1"
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1",
+    "react-dom": "^17.0.2"
```
Comment (oleksandrstarov) [inline package.json:63]: O_o

**Comment by oleksandrstarov** [comment]
Moving react-dom dep to the dev dependencies broke hosting - I wonder why this was done?

**Pull request merged**